src/views/Patient/MedReqDropDown/MedReqDropDown.tsx

```diff
@@ -279,9 +282,11 @@ function MedReqDropDown({
       };
       axios({
         method: 'post',
-        url: `${env
-          .get('REACT_APP_REMS_ADMIN_SERVER_BASE')
-          .asString()}/4_0_0/GuidanceResponse/$rems-etasu`,
+        url: env.get('USE_INTERMEDIARY').asBool()
+          ? `${env.get('INTERMEDIARY_ETASU_MET').asString()}`
```

Comment:
This should return the FHIR URL so the /GuidanceResponse/$rems-etasu can be added to both this and the one below. Could add a function to utils to get the fhir url that would return the intermediary fhir url or the rems admin base + '4_0_0'.